src/application-delegate.coffee

```diff
@@ -126,6 +126,8 @@ class ApplicationDelegate
     loadSettings = getWindowLoadSettings()
     loadSettings['initialPaths'] = paths
     setWindowLoadSettings(loadSettings)
+    ipcRenderer.send("did-save-state")
```

Comment:
I think we should rename the event from `did-save-state` to `did-change-paths`. The current name `did-save-state` makes it sound like Atom already *did* save the state.